api/rosetta/parser.go

```diff
@@ -27,4 +27,5 @@ type Parser interface {
 	DeriveIntent(operations []object.Operation) (intent *transactions.Intent, err error)
 	CompileTransaction(intent *transactions.Intent, metadata object.Metadata) (tx *flow.Transaction, err error)
 	ParseTransaction(tx *flow.Transaction) (operations []object.Operation, signers []identifier.Account, err error)
+	SignTransaction(unsignedTx *flow.Transaction, signature object.Signature) (tx *flow.Transaction, err error)
```

Comment:
I would like to rename this, as it is not really _signing_ the transaction. It's combining the transaction with the signature. Something like `AttachSignature(tx, sig)` would work for me.